packages/nuxt/src/pages/runtime/app.vue

```diff
@@ -3,3 +3,7 @@
     <NuxtPage />
   </NuxtLayout>
 </template>
+
+<script setup>
+import NuxtPage from './page'
+</script>
```

Comment:
Apologies, you were probably right the first time. We don't import `<NuxtLayout>` but rely on the loader here; we can do the same for NuxtPage.

```suggestion

<script setup>
import NuxtPage from './page'
</script>
```

Reply:
nice !